src/status_im/translations/nl.cljs

```diff
@@ -8,11 +8,23 @@
    :chat-name                             "Chatnaam"
    :notifications-title                   "Meldingen en geluiden"
    :offline                               "Offline"
+   :search-for                            "Zoek naar..."
+   :cancel                                "Annuleren"
+   :next                                  "Volgende"
+   :type-a-message                        "Typ een bericht..."
+   :type-a-command                        "Typ een commando..."
+   :error                                 "Error"
+
+   :camera-access-error                   "Om de benodigde cameratoegang te krijgen kan je naar de instellingen gaan en Status > Camera inschakelen."
+   :photos-access-error                   "Om de benodigde galerijtoegang te krijgen kan je naar de instellingen gaan en Status > Foto's inschakelen."
 
    ;drawer
    :invite-friends                        "Nodig vrienden uit"
    :faq                                   "FAQ"
    :switch-users                          "Schakel tussen gebruikers"
+   :feedback                              "Heb je opmerkingen?\nSchud je telefoon!"
```

Comment:
`Heb je feedback?\nSchud je telefoon!`

Feedback is also a word in Dutch see [van Dale](http://www.vandale.nl/opzoeken?pattern=feedback&lang=nn).